BackendApartmentReservation/BackendApartmentReservation/Authentication/SecurityManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
+using BackendApartmentReservation.Infrastructure.Utilities;
+
+namespace BackendApartmentReservation.Authentication
+{
+    public class SecurityManager
```

Comment:
Utility classes are not meant to be instantiated. - Sonar 
Is it possible to add ` static ` or ` protected ` constructor? 😃

Reply:
Uhm I guess so :D will simply add empty static constructor

Reply:
Or maybe make class static